config/Run3_2022/samples.yaml

```diff
@@ -42,3 +36,1092 @@ MET_Run2022C:
+  sampleType: QCD
+
+############## QCD_PT-*to*_MuEnriched ##############
+QCD_PT-15to20_MuEnrichedPt5:
+  crossSection: QCD_PT-15to20_MuEnrichedPt5
+  generator: pythia8
+  miniAOD: /QCD_PT-15to20_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-20to30_MuEnrichedPt5:
+  crossSection: QCD_PT-20to30_MuEnrichedPt5
+  generator: pythia8
+  miniAOD: /QCD_PT-20to30_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-30to50_MuEnrichedPt5:
+  crossSection: QCD_PT-30to50_MuEnrichedPt5
+  generator: pythia8
+  miniAOD: /QCD_PT-30to50_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-50to80_MuEnrichedPt5:
+  crossSection: QCD_PT-50to80_MuEnrichedPt5
+  generator: pythia8
+  miniAOD: /QCD_PT-50to80_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-80to120_MuEnrichedPt5:
+  crossSection: QCD_PT-80to120_MuEnrichedPt5
+  generator: pythia8
+  miniAOD: /QCD_PT-80to120_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-120to170_MuEnrichedPt5:
+  crossSection: QCD_PT-120to170_MuEnrichedPt5
+  generator: pythia8
+  miniAOD: /QCD_PT-120to170_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-170to300_MuEnrichedPt5:
+  crossSection: QCD_PT-170to300_MuEnrichedPt5
+  generator: pythia8
+  miniAOD: /QCD_PT-170to300_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-300to470_MuEnrichedPt5:
+  crossSection: QCD_PT-300to470_MuEnrichedPt5
+  generator: pythia8
+  miniAOD: /QCD_PT-300to470_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-470to600_MuEnrichedPt5:
+  crossSection: QCD_PT-470to600_MuEnrichedPt5
+  generator: pythia8
+  miniAOD: /QCD_PT-470to600_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-600to800_MuEnrichedPt5:
+  crossSection: QCD_PT-600to800_MuEnrichedPt5
+  generator: pythia8
+  miniAOD: /QCD_PT-600to800_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-800to1000_MuEnrichedPt5:
+  crossSection: QCD_PT-800to1000_MuEnrichedPt5
+  generator: pythia8
+  miniAOD: /QCD_PT-800to1000_MuEnrichedPt5_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+o120_EMEnriched_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
```

Comment:
ill formated yaml

Reply:
Yeah I think this was garbage from manual copy and pasting, removed